CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 - **decidim-proposals**: Hide proposal vote button when proposal is answered and rejected. [\#1861](https://github.com/decidim/decidim/pull/1861)
 
+**Changed**
+
+- **decidim**: URLs now use the participatory space slug instead of the URL, both in the public pages and in the admin. Old routes using the space IDs now redirect to the ones using the slug. [\#1842](https://github.com/decidim/decidim/pull/1842)
```

Comment:
Instead of the ID

Reply:
We're not technically redirecting, only serving the same content (except for the participatory process and assembly page where we are indeed redirecting). There's a difference in terms of SEO, so I'd say "Old routes using the space IDs still serve the same content as the ones using the slug".

Reply:
Mh I think it's a proper redirection, sending a 301 HHTP status code:

http://guides.rubyonrails.org/routing.html#redirection

Reply:
I just tried it and it working perfectly! I was probably just misreading the patch, sorry! :blush: